helix-term/src/application.rs

```diff
@@ -471,7 +471,16 @@ impl Application {
                 }
             }
             signal::SIGCONT => {
-                self.claim_term().await.unwrap();
+                // Copy/Paste from same issue from neovim:
+                // https://github.com/neovim/neovim/issues/12322
+                let mut retry_count = 10;
+                while retry_count > 0 {
```

Comment:
Instead of copy-pasting this should probably be made more Rusty, something along the lines of
```rs
for _ in 0..10 {
    if let Ok(_) = self.claim_term().await {
        break;
    }
}
```